components/dashboard/src/projects/new-project/NewProjectCreateFromURL.tsx

```diff
@@ -59,14 +59,14 @@ export const NewProjectCreateFromURL: FC<Props> = ({ repoSearchFilter, isCreatin
             // Repo is last segment
             const repo = segments.pop();
             // owner is everything else
-            const owner = segments.join("/");
+            const owner = segments.join("-");
 
-            if (!owner && !repo) {
+            if (!repo) {
                 throw new Error();
             }
 
-            name = repo || owner;
-            slug = (repo || owner).toLowerCase();
+            name = repo;
+            slug = [repo, owner].filter(Boolean).join("-").toLowerCase();
```

Comment:
just to resurface: this won't work nice with personal BBS repos, e.g. `https://bitbucket.gitpod-dev.com/scm/~alex/personal-repo.git`

Reply:
I ended up swapping it so it's repo at the end. So in this case the slug would be `scm-~alex-personal-repo` - is that a problem?